rust/lib/Cargo.toml

```diff
@@ -7,5 +7,6 @@ edition = "2021"
 [dependencies]
 zingolib = { workspace = true }
 zingoconfig = { workspace = true }
+http = "*"
```

Comment:
That's good, with this you solve this other problem: the error: `get_lightd_info response at https://mainnet.lightwalletd.com:9067/: Status { code: Unknown, message: "request has unsupported HTTP version", source: Some(hyper::Error(User(UnsupportedVersion))) }`